frontend/src/static/css/style.scss

```diff
@@ -45,6 +45,11 @@ a{
     }
 }
 
+.mini-font{
+  font-size:.65rem;
```

Comment:
I think it is better for you to indent 4 spaces instead of 2 because the code style of PR #803 is not confirmed yet. If PR #803 merges before this, then you can resolve the conflicts.